webapp/app/styles/app.css.map

```diff
@@ -0,0 +1,7 @@
sCAAO;MACL,KAAK,E/C2sBmB,OAAiB;M+C1sBzC,gBAAgB,E/CwsBQ,OAAmB;E+CpsB7C,8DAAgC;IAC9B,mBAAmB,E/CosBK,OAAqB;;A8CtdnD,aAAc;EC/PZ,YAAY,E/CytBgB,OAAoB;E+CvtBhD,8BAAmB;IACjB,KAAK,E/CqtBqB,OAAkB;I+CptB5C,gBAAgB,E/CstBU,OAAgB;I+CrtB1C,YAAY,E/CotBc,OAAoB;I+CltB9C,8DAAgC;MAC9B,gBAAgB,E/CitBQ,OAAoB;I+C/sB9C,qCAAO;MACL,KAAK,E/C+sBmB,OAAgB;M+C9sBxC,gBAAgB,E/C4sBQ,OAAkB;E+CxsB5C,6DAAgC;IAC9B,mBAAmB,E/CwsBK,OAAoB;;AgDxtBlD,iBAAkB;EAChB,QAAQ,EAAE,QAAQ;EAClB,OAAO,EAAE,KAAK;EACd,MAAM,EAAE,CAAC;EACT,OAAO,EAAE,CAAC;EACV,QAAQ,EAAE,MAAM;EAEhB;;;;yBAIM;IACJ,QAAQ,EAAE,QAAQ;IAClB,GAAG,EAAE,CAAC;IACN,IAAI,EAAE,CAAC;IACP,MAAM,EAAE,CAAC;IACT,MAAM,EAAE,IAAI;IACZ,KAAK,EAAE,IAAI;IACX,MAAM,EAAE,CAAC;EAIX,wCAAyB;IACvB,cAAc,EAAE,MAAM;EAIxB,uCAAwB;IACtB,cAAc,EAAE,GAAG;;AC1BvB,KAAM;EACJ,UAAU,EAAE,IAAI;EAChB,OAAO,EAAE,IAAI;EACb,aAAa,EAAE,IAAI;EACnB,gBAAgB,EjD6uBY,OAAO;EiD5uBnC,MAAM,EAAE,iBAAsB;EAC9B,aAAa,EjDoGa,GAAG;ED5C7B,kBAAkB,EAAE,mCAAO;EACnB,UAAU,EAAE,mCAAO;EkDvD3B,gBAAW;IACT,YAAY,EAAE,IAAI;IAClB,YAAY,EAAE,mBAAe;;AAKjC,QAAS;EACP,OAAO,EAAE,IAAI;EACb,aAAa,EjD0Fa,GAAG;;AiDxF/B,QAAS;EACP,OAAO,EAAE,GAAG;EACZ,aAAa,EjDuFa,GAAG;;AkD7G/B,MAAO;EACL,KAAK,EAAE,KAAK;EACZ,SAAS,EAAE,IAAuB;EAClC,WAAW,ElD2yBiB,IAAI;EkD1yBhC,WAAW,EAAE,CAAC;EACd,KAAK,ElD0yBuB,IAAI;EkDzyBhC,WAAW,ElD0yBiB,YAAa;EkBlzBzC,OAAO,EgCSU,GAAE;EhCNnB,MAAM,EAAE,iBAA6B;EgCQrC,0BACQ;IACN,KAAK,ElDoyBqB,IAAI;IkDnyB9B,eAAe,EAAE,IAAI;IACrB,MAAM,EAAE,OAAO;IhCfjB,OAAO,EgCgBY,GAAE;IhCbrB,MAAM,EAAE,iBAA6B;;AgCsBvC,YAAa;EACX,OAAO,EAAE,CAAC;EACV,MAAM,EAAE,OAAO;EACf,UAAU,EAAE,WAAW;EACvB,MAAM,EAAE,CAAC;EACT,kBAAkB,EAAE,IAAI;;ACvB1B,WAAY;EACV,QAAQ,EAAE,MAAM;;AAIlB,MAAO;EACL,OAAO,EAAE,IAAI;EACb,QAAQ,EAAE,MAAM;EAChB,QAAQ,EAAE,KAAK;EACf,GAAG,EAAE,CAAC;EACN,KAAK,EAAE,CAAC;EACR,MAAM,EAAE,CAAC;EACT,IAAI,EAAE,CAAC;EACP,OAAO,EnD4PkB,IAAI;EmD3P7B,0BAA0B,EAAE,KAAK;EAIjC,OAAO,EAAE,CAAC;EAGV,yBAAqB;IpD0GrB,iBAAiB,EAAE,kBAAiB;IAChC,aAAa,EAAE,kBAAiB;IAC/B,YAAY,EAAE,kBAAiB;IAC5B,SAAS,EAAE,kBAAiB;IAkEpC,kBAAkB,EAAE,+BAA6B;IAC9C,eAAe,EAAE,4BAA0B;IACzC,aAAa,EAAE,0BAAwB;IACpC,UAAU,EAAE,uBAAqB;EoD9KzC,uBAAmB;IpDsGnB,iBAAiB,EAAE,eAAiB;IAChC,aAAa,EAAE,eAAiB;IAC/B,YAAY,EAAE,eAAiB;IAC5B,SAAS,EAAE,eAAiB;;AoDvGtC,kBAAmB;EACjB,UAAU,EAAE,MAAM;EAClB,UAAU,EAAE,IAAI;;AAIlB,aAAc;EACZ,QAAQ,EAAE,QAAQ;EAClB,KAAK,EAAE,IAAI;EACX,MAAM,EAAE,IAAI;;AAId,cAAe;EACb,QAAQ,EAAE,QAAQ;EAClB,gBAAgB,EnD+hB6B,IAAI;EmD9hBjD,MAAM,EAAE,cAA8C;EACtD,MAAM,EAAE,4BAAqC;EAC7C,aAAa,EnD0Da,GAAG;ED7C7B,kBAAkB,EAAE,4BAAO;EACnB,UAAU,EAAE,4BAAO;EoDZ3B,eAAe,EAAE,WAAW;EAE5B,OAAO,EAAE,CAAC;;AAIZ,eAAgB;EACd,QAAQ,EAAE,QAAQ;EAClB,GAAG,EAAE,CAAC;EACN,KAAK,EAAE,CAAC;EACR,IAAI,EAAE,CAAC;EACP,gBAAgB,EnDshBY,IAAI;EmDphBhC,oBAAO;IjCnEP,OAAO,EiCmEmB,CAAC;IjChE3B,MAAM,EAAE,gBAA6B;EiCiErC,kBAAK;IjCpEL,OAAO,ElBylBqB,GAAE;IkBtlB9B,MAAM,EAAE,iBAA6B;;AiCsEvC,aAAc;EACZ,OAAO,EnDigBqB,IAAI;EmDhgBhC,aAAa,EAAE,iBAAoC;EACnD,UAAU,EAAE,UAAiD;;AAG/D,oBAAqB;EACnB,UAAU,EAAE,IAAI;;AAIlB,YAAa;EACX,MAAM,EAAE,CAAC;EACT,WAAW,EnDufiB,OAAiB;;AmDlf/C,WAAY;EACV,QAAQ,EAAE,QAAQ;EAClB,OAAO,EnD2eqB,IAAI;;AmDvelC,aAAc;EACZ,OAAO,EnDseqB,IAAI;EmDrehC,
```

Comment:
Source maps should not be committed - they should be in the ignore list, since they change all the time... Same goes for result CSS files (they're build products, not source files)